app-modules/engagement/src/Jobs/ProcessSesS3InboundEmail.php

```diff
@@ -170,14 +169,7 @@ public function handle(): void
                         ->where('email', $sender)
                         ->get();
 
-                    if (! UnMatchInboundCommunicationFeature::active()) {
-                        throw_if(
-                            $contacts->isEmpty(),
-                            new UnableToDetectAnyMatchingContactsFromSesS3EmailPayload($this->emailFilePath),
```

Comment:
Is this exception being used anywhere any more after this? If not, let's delete it.